packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1427,33 +1433,16 @@ class StackViewport extends Viewport implements IStackViewport {
     bitsAllocated,
     numComps,
     numVoxels,
-    TypedArray,
+    typedArray,
```

Comment:
typedArray is the type of the thing, not what it is in terms of usage - please make this one pixelArray, as it is an array of pixels that happens to be of type TypedArray for general usage.